src/components/Text.tsx

```diff
@@ -1,10 +1,24 @@
-/* eslint-disable camelcase */
-import React from 'react';
+import React, {FC, ReactNode} from 'react';
 import PropTypes from 'prop-types';
 import chalk from 'chalk';
 
-const Text = ({bold, italic, underline, strikethrough, children, unstable__transformChildren}) => {
-	const transformChildren = children => {
+interface TextProps {
+	readonly bold?: boolean;
+	readonly italic?: boolean;
+	readonly underline?: boolean;
+	readonly strikethrough?: boolean;
+	readonly unstable__transformChildren?: (x: string) => string;
```

Comment:
Don't use one character variable names. Use descriptive names.

Reply:
Changed to `children`